src/adapters/sqlite/encodeSchema/test.js

```diff
@@ -54,6 +54,34 @@ describe('encodeSchema', () => {
 
     expect(encodeSchema(testSchema)).toBe(expectedSchema)
   })
+  it('encodes schema with FTS', () => {
+    const testSchema = appSchema({
+      version: 1,
+      tables: [
+        tableSchema({
+          name: 'tasks',
+          columns: [
+            { name: 'author_id', type: 'string', isIndexed: true },
+            { name: 'author_name', type: 'string', isSearchable: true },
+            { name: 'author_title', type: 'string', isSearchable: true },
+            { name: 'created_at', type: 'number' },
+          ],
+        }),
+      ],
+    })
+
+    const expectedSchema =
+      'create table "tasks" ("id" primary key, "_changed", "_status", "author_id", "author_name", "author_title", "created_at");' +
+      'create index tasks_author_id on "tasks" ("author_id");' +
+      'create index tasks__status on "tasks" ("_status");' +
+      'create virtual table "tasks_fts" using fts4("author_name", "author_title");' +
```

Comment:
let's rename the virtual tables and triggers so that they start with `_fts_NAME` - this is to ensure that there won't be any accidental name conflicts (someone could name a table `tasks_fts`, but a table starting with `_` will fail validation IIRC - pls verify)